setup.py

```diff
@@ -20,6 +25,20 @@ def get_description():
     'freezegun'
 ]
 
+# TODO: let a flag or option determine whether to actually install these
+install_data = [
```

Comment:
Yeah, watch out for this.  This makes things *really* screwy when you try to pip install the project into a virtualenv.  We've tried doing it in a handful of other projects and always ended up backing it out and relying on .spec files to handle `/usr/share` and `/etc`.